src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,93 @@
+import calculate from '../components/Calculator/logic/calculate';
+
+describe('calculate()', () => {
+  it('returns an object with null properties when the AC button is pressed', () => {
+    const button = 'AC';
+    const obj = {
+      total: 5,
+      next: 5,
+      operation: '+',
+    };
+
+    const expected = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+    const result = calculate(obj, button);
+
+    expect(result).toStrictEqual(expected);
+  });
+
+  it('should update the total property for the = button', () => {
+    const button = '=';
+    const obj = {
+      total: 5,
+      next: 5,
+      operation: '+',
+    };
+
+    const expected = {
```

Comment:
- [ ] I notice you completed snapshot testing, however when I execute the npm test command in the vscode terminal, I get the problem shown in the attachment. :point_down: 
![Screenshot from 2022-12-08 15-28-49](https://user-images.githubusercontent.com/88894525/206458533-9a92304a-0f39-4023-8704-33786755c24a.png)
 
I was not able to run your project on my local machine because of that error but I tried to run this command and it really helps, I suggest you have a look at [this](https://stackoverflow.com/questions/70204039/failed-to-initialize-watch-plugin-node-modules-jest-watch-typeahead-filename-js) documentation it will help you to run your test locally :wink: